docs/index.rst

```diff
@@ -1,14 +1,100 @@
+
+A schematic of the above is shown in :numref:`Schematic_Layout_Unified_Workflow_Framework` below.
+
+.. _Schematic_Layout_Unified_Workflow_Framework:
+	
+.. figure:: /static/figure/Schematic_Layout_Unified_Workflow_Framework.png
+
+   *Schematic layout of the various components of the Unified Workflow Framework.*
+
+The tools on the right are developed as stand-alone packages that can be deployed independently on different platforms (like libraries). On the left are configuration/schema files that provide the default configurations for the workflows. These can change as the modeling systems develop. In the middle are the workflow layers that serve different needs.
+
+Using the tools on the right, any application can assemble a unique suite for its needs. This allows all applications to retain similar implementations while at the same time providing a mechanism to construct their own flavor of the implementation. 
+
+.. _UWStructure:
+
+Directory Structure
+======================
+The ``workflow-tools`` :term:`repository` is an NCO-compliant repository. Its structure follows the standards laid out in :term:`NCEP` Central Operations (NCO) WCOSS `Implementation Standards <https://www.nco.ncep.noaa.gov/idsb/implementation_standards/ImplementationStandards.v11.0.0.pdf?>`__. 
+
+.. code-block:: console
+
+   workflow-tools
+   ├── docs 
+   │     └── sections
+   │           ├── contributor_guide
+   │           └── user_guide
+   └── src
+         └── (uwtools)
+    	         ├── api
+    	         ├── apps 
+    	         ├── config
+               │     └── formats
+    	         ├── drivers 
+    	         ├── files
+               │     ├── gateway
+               │     ├── interface
+               │     └── model
+    	         ├── resources
+    	         ├── tests
+               │     ├── api
+               │     ├── config
+               │     │     └── formats
+               │     ├── drivers
+               │     ├── files
+               │     │     ├── gateway
+               │     │     ├── interface
+               │     │     └── model
+               │     ├── fixtures
+               │     │     └── files
```

Comment:
I would take a look again at this table. Maybe removing the `|` to the left of nested items will help this format better.

Confirming I am looking at this documentation: https://unified-workflow--366.org.readthedocs.build/en/366/#